### ISSUE TITLE
Add Accessibility Features

### ISSUE THREAD

**Issue by samuel-obrien25** [defund12/defund12.org#332]
### Main

- Change div.container to main.container

- "12" = 🚓 surrounding p element should have an aria-label="12 equals police"

- Add aria-labels to H2s

- Copy to clipboard spans should be buttons with aria-label="select to copy {Link Text} to clipboard"

### When City is Selected

- Send Email and Copy Link anchors do not have hrefs, therefore are not accessible through keyboard navigation. Can be fixed by implementing href='javascript:void(0)'
